runner/src/check_todo.rs

```diff
@@ -15,16 +15,48 @@
 //
 
 use super::*;
+use regex::Regex;
+use std::cmp::max;
 
 /// A [`Runnable`] command that checks for the existence of todos in the codebase with no associated
```

Comment:
Update the doc to mention that it only looks for todos on commented lines

Reply:
Just curious, why is the mutex needed here?

Reply:
Too much copy pasting! Sorry! It is not needed!